### PR TITLE
4_4_6_Max_in_place

### PR DESCRIPTION
**Pull request by AleksandraYanum**

### CODE REVIEW

File: 4_Lists/4_4_Matrix_Part_1/6_Max_in_place.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+            current_max_num = max([matrix[row][col],
+                                   matrix[row][matrix_size - col - 1],
+                                   matrix[matrix_size - row - 1][matrix_size - col - 1],
+                                   matrix[matrix_size - row - 1][col]
+                                   ])
+            max_matrix_num = max(max_matrix_num, current_max_num)
+
+    print(max_matrix_num)
+
+
```
Comment (YanumVlad) [inline 4_Lists/4_4_Matrix_Part_1/6_Max_in_place.py:18]: Add comment

Comment (AleksandraYanum) [reply 4_Lists/4_4_Matrix_Part_1/6_Max_in_place.py:18]: Done

**Review by YanumVlad — APPROVED** [review]

**Comment by YanumVlad** [comment]
Pls reply: PING

**Review by YanumVlad — COMMENTED** [review]
Reply?

**Review by YanumVlad — CHANGES_REQUESTED** [review]
request change

**Review by YanumVlad — APPROVED** [review]

**Review by YanumVlad — APPROVED** [review]

**Pull request merged**